From 9dfcae6cf5d09d52d1135a38fce5199dd2350cbc Mon Sep 17 00:00:00 2001
From: Ganesan Ramalingam <ganesanr@broadcom.com>
Date: Thu, 6 Mar 2014 08:00:53 +0530
Subject: [PATCH 0516/1532] GPIO: XLP5xx Lite Eval Board uses GPIO 53 as SMSC
 interrupt pin

GPIO 53 is used for SMSC interrupt pin in XLP5xx Lite boards
[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/mips/boot/dts/xlp5xx-linux-tmpl.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/boot/dts/xlp5xx-linux-tmpl.dts b/arch/mips/boot/dts/xlp5xx-linux-tmpl.dts
index 93b20bb..b5a6517 100644
--- a/arch/mips/boot/dts/xlp5xx-linux-tmpl.dts
+++ b/arch/mips/boot/dts/xlp5xx-linux-tmpl.dts
@@ -228,7 +228,7 @@
 		        reg = <1 0x01200000 0x100000>;
 			reg-io-width = <4>;
 
-		        interrupts = <39>;
+		        interrupts = <53>;
 		        interrupt-parent = <&gpio>;
 
 		        phy-mode = "mii";
-- 
1.9.1

